sampleproject/components/calendar/calendar.py

```diff
@@ -8,9 +8,9 @@ class Calendar(Component):
     #
     # `template_name` can be relative to dir where `calendar.py` is, or relative to COMPONENTS.dirs
     template_name = "calendar/calendar.html"
-    # Or
-    # def get_template_name(context):
-    #     return f"template-{context['name']}.html"
+
+    css_file = "calendar/calendar.css"
+    js = "calendar/calendar.js"
```

Comment:
js_file, why doesn't this break?

Reply:
I did actually break, thanks for spotting that :D